### DISCUSSION TITLE
If I want to create driver data for more than a year, how should I modify it.

### DISCUSSION THREAD

**Discussion by FuYingWang** [NCAR/hrldas#139] [Q&A]
When I used 'extract_GLDAS.py', I found that it requires entering a specific day in a year as the start and end. If my data is 2 years or more, how should I modify it. Or is it that this code can only obtain data for one year？
![extract_GLDAS](https://github.com/NCAR/hrldas/assets/153582912/fb663354-3260-4962-b48c-f86caa5ee620)

**Comment by cenlinhe** [comment]
I think current script can only handle the data one year by one year. You will have to run the script for this year first, and then modify and run the script for the second year. We will try to change the script to be more flexible to handle multiple years at the same time in the near future.
